solidity/test/LPRewardsTest.js

```diff
@@ -0,0 +1,220 @@
+const {contract, web3, accounts} = require("@openzeppelin/test-environment")
+const {createSnapshot, restoreSnapshot} = require("./helpers/snapshot")
+const {time} = require("@openzeppelin/test-helpers")
+
+const LPRewards = contract.fromArtifact("LPRewards")
+const KeepToken = contract.fromArtifact("KeepToken")
+const WrappedToken = contract.fromArtifact("TestToken")
+
+const BN = web3.utils.BN
+const chai = require("chai")
+chai.use(require("bn-chai")(BN))
+const expect = chai.expect
+
+describe("LPRewards", () => {
+  const tokenDecimalMultiplier = web3.utils.toBN(10).pow(web3.utils.toBN(18))
+
+  let keepToken
+  let lpRewards
+  let wrappedToken
+  let owner
+  let wallet1
+  let wallet2
+
+  before(async () => {
+    owner = accounts[0]
```

Comment:
Thoughts on naming it `rewardDistribution` as it is in the contract.

Reply:
044bcab